Misc/NEWS.d/next/Windows/2019-04-04-08-59-44.bpo-9194.DzakhD.rst

```diff
@@ -0,0 +1 @@
+Fix the bounds checking in winreg.c's ``fixupMultiSZ()``.
```

Comment:
NEWs entry isn't needed for trivial fixed. I already set the 'skip news' label.

Reply:
@berkerpeksag I've removed it.